pkgs/applications/audio/spotify/default.nix

```diff
@@ -87,7 +87,14 @@ stdenv.mkDerivation {
       # Desktop file
       mkdir -p "$out/share/applications/"
       cp "$out/share/spotify/spotify.desktop" "$out/share/applications/"
-      sed -i "s|Icon=.*|Icon=$out/share/spotify/Icons/spotify-linux-512.png|" "$out/share/applications/spotify.desktop"
+
+      # Icons
+      for i in 16 22 24 32 48 64 128 256 512; do
+        ixi="$i"x"$i"
+        mkdir -p "$out/share/icons/hicolor/$ixi/apps"
+        cp "$out/share/spotify/icons/spotify-linux-$i.png" \
```

Comment:
Is there any reason not to use symbolic links here? (I realize the icons are small.)